packages/replayio/src/utils/mixpanel/initMixpanelForUserSession.ts

```diff
@@ -0,0 +1,25 @@
+import { getUserIdOrThrow } from "../graphql/getUserIdOrThrow";
+import { debug } from "./debug";
+import { configureSession } from "./session";
+import { getMixpanelAPI } from "./getMixpanelAPI";
+
+export async function initMixpanelForUserSession(accessToken: string | undefined) {
+  const mixpanelAPI = getMixpanelAPI();
+  if (!mixpanelAPI) {
+    return;
```

Comment:
I feel like we should call `configureSession` here - otherwise we are risking that some `await` hangs on `deferredSession.promise`.
```suggestion
    configureSession(undefined);
    return;
```

This is probably unlikely today though cause all of the callers to `trackEventAsync` should bail out early on `!mixpanelAPI` too